tide/core/ScreenLock.h

```diff
@@ -78,20 +78,20 @@ class ScreenLock : public QObject,
     /** Check the state of the lock */
     bool isLocked();
 
-    /** Close the specified stream. */
-    Q_INVOKABLE void closeStream(QString uri);
+    /** Reject the specified stream. */
+    Q_INVOKABLE void rejectStream(QString uri);
 
-    /** Release the specified stream. */
-    Q_INVOKABLE void releaseStream(QString uri);
+    /** Accept the specified stream. */
+    Q_INVOKABLE void acceptStream(QString uri);
 
-    /** Deregister the specified stream from the lock. */
-    void deregisterStream(QString uri);
+    /** Cancel acceptance of the specified stream. */
+    void cancelStreamAcceptance(QString uri);
 
-    /** Register a new stream to the lock. */
-    void registerStream(QString uri);
+    /** Request a new stream acceptance. */
+    void requestStreamAcceptance(QString uri);
```

Comment:
consider reordering the methods in the logical order they can be used, with the "positive" action first:

requestStreamAcceptance
cancelStreamAcceptance

acceptStream
rejectStream